10.0.26100.0 build isn't working

## Context

Workflow returns an error when running a pipeline to build the new 26100 SDK

Workflow run: https://github.com/GuillaumeFalourd/setup-windows10-sdk-action/actions/runs/10254371966/job/28370159382

Workflow config:

```
  specific-sdk-version-26100:
    runs-on: windows-latest
    steps:
      - uses: actions/checkout@v2.3.4
      - uses: GuillaumeFalourd/setup-windows10-sdk-action@main
        with:
          sdk-version: 26100

      - name: Build new SDK
        shell: bash
        run: |
          cmake -DCMAKE_SYSTEM_VERSION=10.0.26100.0
          cmake --build .
  ```
## Logs

Executing:
```
Run cmake -DCMAKE_SYSTEM_VERSION=10.0.26100.0
  cmake -DCMAKE_SYSTEM_VERSION=10.0.[2](https://github.com/GuillaumeFalourd/setup-windows10-sdk-action/actions/runs/10254371966/job/28370159382#step:4:2)6100.0
  cmake --build .
  shell: C:\Program Files\Git\bin\bash.EXE --noprofile --norc -e -o pipefail {0}
```
The returned error is:
```
 CMake Warning:
  No source or binary directory provided.  Both will be assumed to be the
  same as the current working directory, but note that this warning will
  become a fatal error in future CMake releases.
-- Building for: Visual Studio 17 2022
CMake Error at CMakeLists.txt:5 (project):
  Failed to run MSBuild command:
    C:/Program Files/Microsoft Visual Studio/2022/Enterprise/MSBuild/Current/Bin/amd64/MSBuild.exe
  to get the value of VCTargetsPath:
    MSBuild version 17.10.4+10fbfbf2e for .NET Framework
    Build started 8/5/2024 7:22:59 PM.
    
    Project "D:\a\setup-windows10-sdk-action\setup-windows10-sdk-action\CMakeFiles\[3](https://github.com/GuillaumeFalourd/setup-windows10-sdk-action/actions/runs/10254371966/job/28370159382#step:4:3).30.1\VCTargetsPath.vcxproj" on node 1 (default targets).
    C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.Cpp.WindowsSDK.targets([4](https://github.com/GuillaumeFalourd/setup-windows10-sdk-action/actions/runs/10254371966/job/28370159382#step:4:4)9,5): error MSB8037: The Windows SDK version 10.0.26100.0 for Desktop C++ x64 Apps was not found. Install the required version of Windows SDK or change the SDK version in the project property pages or by right-clicking the solution and selecting "Retarget solution".  [D:\a\setup-windows10-sdk-action\setup-windows10-sdk-action\CMakeFiles\3.30.1\VCTargetsPath.vcxproj]
    Done Building Project "D:\a\setup-windows10-sdk-action\setup-windows10-sdk-action\CMakeFiles\3.30.1\VCTargetsPath.vcxproj" (default targets) -- FAILED.
    
    Build FAILED.
    
    "D:\a\setup-windows10-sdk-action\setup-windows10-sdk-action\CMakeFiles\3.30.1\VCTargetsPath.vcxproj" (default target) (1) ->
    (_CheckWindowsSDKInstalled target) -> 
      C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.Cpp.WindowsSDK.targets(49,[5](https://github.com/GuillaumeFalourd/setup-windows10-sdk-action/actions/runs/10254371966/job/28370159382#step:4:5)): error MSB8037: The Windows SDK version 10.0.2[6](https://github.com/GuillaumeFalourd/setup-windows10-sdk-action/actions/runs/10254371966/job/28370159382#step:4:7)100.0 for Desktop C++ x64 Apps was not found. Install the required version of Windows SDK or change the SDK version in the project property pages or by right-clicking the solution and selecting "Retarget solution".  [D:\a\setup-windows10-sdk-action\setup-windows10-sdk-action\CMakeFiles\3.30.1\VCTargetsPath.vcxproj]
    
        0 Warning(s)
        1 Error(s)
    
    Time Elapsed 00:00:13.6[8](https://github.com/GuillaumeFalourd/setup-windows10-sdk-action/actions/runs/10254371966/job/28370159382#step:4:9)
    
  Exit code: 1
-- Configuring incomplete, errors occurred!
Error: Process completed with exit code 1.
```

## References

Related PR: https://github.com/GuillaumeFalourd/setup-windows10-sdk-action/pull/22

--- a/externals/install-winsdk.ps1
+++ b/externals/install-winsdk.ps1
@@ -49,7 +49,7 @@ function Download-File {
     }
 
     if ($buildNumber -eq "20348") {
-        $downloadUrl = "https://software-download.microsoft.com/download/pr/20348.1.210507-1500.fe_release_WindowsSDK.iso"
+        $downloadUrl = "https://go.microsoft.com/fwlink/?linkid=2164360"
     }
 
     if ($buildNumber -eq "22000") {
@@ -61,7 +61,7 @@ function Download-File {
     }
 
     if ($buildNumber -eq "26100") {
-        $downloadUrl = "https://software-static.download.prss.microsoft.com/dbazure/888969d5-f34g-4e03-ac9d-1f9786c66749/26100.1.240331-1435.ge_release_WindowsSDK.iso"
+        $downloadUrl = "https://go.microsoft.com/fwlink/?linkid=2286663"
     }
 
     try {

